hudi-common/src/main/java/org/apache/hudi/common/model/EventTimeAvroPayload.java

```diff
@@ -41,7 +42,7 @@ public EventTimeAvroPayload(GenericRecord record, Comparable orderingVal) {
   }
 
   public EventTimeAvroPayload(Option<GenericRecord> record) {
-    this(record.isPresent() ? record.get() : null, 0); // natural order
+    this(record.isPresent() ? record.get() : null, COMMIT_TIME_ORDERING_VALUE); // natural order
```

Comment:
so, in general we assume integer as default even if ordering col chosen is string or other data types? 
and we never broke anything so far is it?